nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -551,11 +563,19 @@ def _get_metrics_calculator(self) -> MetricsCalculator:
 
 
 class ActivationAPoZRankPruner(ActivationPruner):
+    def _activation_trans(self, output: Tensor) -> Tensor:
+        # return the sum of zero counts of the same activation position cross each batch.
```

Comment:
seems this comment is not what this function does?

Reply:
BTW, where is `self._activation` defined?

Reply:
updated the comment, `self._activation` is defined in line 507.